### ISSUE TITLE
`multipass stop` is not idempotent

### ISSUE THREAD

**Issue by ricab** [canonical/multipass#525]
It assumes the instance is running and fails to connect:

```sh
$ multipass list
Name                    State             IPv4             Release
pliable-greyhound       STOPPED           --               Ubuntu 16.04 LTS
$ multipass stop pliable-greyhound
stop failed: cannot connect to the multipass socket                             
Please ensure multipassd is running and '/run/multipass_socket' is accessible
```

**Comment by townsend2010** [comment]
That is a really weird error, almost as if multipassd went away.  I think something else is going on here...

**Comment by ricab** [comment]
Yeah, this message is coming out a lot when there is an error now. I just opened another issue for that.

**Comment by ricab** [comment]
#527

**Comment by townsend2010** [comment]
Ok, I found the issue for this as I was investigating #527.  Will push a PR shortly.

**Comment by townsend2010** [comment]
Closed by #535.
